The-SMIRNOFF-force-field-format.md

```diff
@@ -475,10 +475,42 @@ U = \sum_{i=1}^N k_i * (1 + cos(periodicity_i * phi - phase_i))
 
 If the `potential` attribute is omitted, it defaults to `k*(1+cos(periodicity*theta-phase))`.
 
-|  ProperTorsions section tag version | Tag attributes and default values                                        | Required parameter attributes         | Optional parameter attributes |
-|-------------------------------------|--------------------------------------------------------------------------|---------------------------------------|-------------------------------|
-| 0.3                                 | `potential="k*(1+cos(periodicity*theta-phase))"`, `default_idivf="auto"` | `smirks`, `k`, `phase`, `periodicity` | `idivf`, `id`, `parent_id`    |
 
+#### Fractional torsion bond orders (EXPERIMENTAL)
+
+.. warning:: This functionality is not yet implemented and will appear in a future version of the toolkit.
+
+Fractional torsion bond orders can be used to allow interpolation of torsion parameters.
+This is similar to the functionality provided by fractional bond orders detailed above.
+For example, these parameters:
+```XML
+<ProperTorsions version="0.3" potential="k*(1+cos(periodicity*theta-phase))" default_idivf="auto">
+    <Proper smirks="[*:1]:[#6X4:2]-[#6X4:3]:[*:4]" periodicity1="2" phase1="0.0 * degree" k1="1.00*kilocalories_per_mole" idivf1="1.0"/>
+    <Proper smirks="[*:1]:[#6X4:2]:[#6X4:3]:[*:4]" periodicity1="2" phase1="0.0 * degree" k1="1.40*kilocalories_per_mole" idivf1="1.0"/>
+    ...
+```
+can be replaced by a single parameter line by first invoking the `fractional_bondorder_method` attribute to specify a method for computing the fractional bond order and `fractional_bondorder_interpolation` for specifying the procedure for interpolating parameters between specified integral bond orders:
+```XML
+<ProperTorsions version="0.3" potential="k*(1+cos(periodicity*theta-phase))" default_idivf="auto" fractional_bondorder_method="AM1-Wiberg" fractional_bondorder_interpolation="linear">
+    <Proper smirks="[*:1]:[#6X4:2]~[#6X4:3]:[*:4]" periodicity1="2" phase1="0.0 * degree" k1_bondorder1="1.00*kilocalories_per_mole" k1_bondorder2="1.80*kilocalories_per_mole" idivf1="1.0"/>
+    ...
+```
+This allows specification of the barrier height for e.g. bond orders 1 and 2, and then interpolation between those based on the partial/fractional bond order.
+Note that in actual usage partial/fractional bond order may never be exactly 1 or 2, or perhaps even near 2; these values only serve to define the slope of the line used for interpolation.
+In the example above, we replaced the two proper torsion terms (one single central bond (`-`) and one aromatic central bond (`:`)) with a single term giving the barrier heights for bond order 1 and 2.
```

Comment:
How is this as an alternative?

> This allows specification of the barrier height for e.g. bond orders 1 and 2 (single and double bonds), and then interpolation between those based on the partial/fractional bond order.
Note that in actual usage partial/fractional bond order may never be exactly 1 or 2, or perhaps even near 2; these values only serve to define the slope of the line used for interpolation.
In the example above, we replaced the two proper torsion terms (one single central bond (`-`) and one double central bond (`=`)) with a single term giving the barrier heights for bond order 1 and 2.
If there are cases where the fractional bond order is 1.5, this can correspond to e.g. an aromatic bond.

Reply:
Looks great. Thanks!